v2/shim_js/tests/faiss.test.ts

```diff
@@ -0,0 +1,56 @@
+import { describe, it, expect } from "vitest";
+
+import { init_faiss_index_wrapper } from "../src/index";
+
+describe("FaissIndexWrapper", async () => {
+  it("Basic Operations", async () => {
+    const vs = await init_faiss_index_wrapper({
+      dimension: 10,
+      description: "IDMap2,Flat",
+      metric: "L2",
+    });
+
+    expect(vs.get_metric_type()).to.be.equal("L2");
+    expect(vs.get_dimension()).to.be.equal(10);
+
+    // add two vectors
+    vs.add_vectors_with_ids(
+      new Float32Array([
+        0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0,
+      ]),
+      2,
+      new BigInt64Array([0n, 1n])
+    );
+    expect(vs.get_ntotal()).to.be.equal(2);
```

Comment:
```suggestion
    expect(vs.get_ntotal()).to.be.equal(2n);
```
`vs.get_ntotal()` returns `bigint` so it needs to be `2n`.